### PR TITLE
travis setup

### PR DESCRIPTION
**Pull request by piyush6348**

### CODE REVIEW

**Comment by piyush6348** [comment]
#2

**Comment by m-murad** [comment]
@piyush6348 
Should i do this?

**Comment by piyush6348** [comment]
@m-murad  license issue is there for Constraint Layout?

**Comment by m-murad** [comment]
@piyush6348 
You have added a lot of unnecessary stuff. Please remove that. Especially the emulator and image stuff.

**Comment by m-murad** [comment]
Also the command you are running is `bash -c ./gradlew connectedCheck`
This will check if the emulator is connected, it will not build the app. You can check the travis logs of your latest build and see the app is not being build.

**Comment by m-murad** [comment]
Please check the following travis file I have written - https://github.com/m-murad/cadasta-android-client/blob/develop/.travis.yml

Make a similar file, clean, precise and no extra stuff.

File: .travis.yml [hunk 1]
```diff
@@ -0,0 +1,58 @@
...
+sudo: required
+group: stable
+dist: precise
+jdk: oraclejdk8
+env:
+  global:
+    - GRADLE_OPTS="-Xmx512m"
+    - BUILD_TOOLS="26.0.2"
+    - TARGET_SDK=26
+  matrix:
```
[the remark this answers is not in the document]
Comment (piyush6348) [reply .travis.yml]: aren't we going to add tests to our app in future?

**Comment by garg-anuj** [comment]
Thanks @piyush6348 🎉  🎉  🎉

**Pull request merged**